static/app/views/eventsV2/table/columnEditModal.spec.jsx

```diff
@@ -354,24 +483,25 @@ describe('EventsV2 -> ColumnEditModal', function () {
         },
         initialData
       );
-      expect(newWrapper.find('QueryField ArithmeticError')).toHaveLength(1);
-      expect(newWrapper.find('QueryField ArithmeticError').prop('title')).toBe(
-        'Division by 0 is not allowed'
-      );
 
-      const field = newWrapper.find('QueryField input[type="text"]');
-      changeInputValue(field, '1+1+1+1+1+1+1+1+1+1+1+1');
-      newWrapper.update();
-      field.simulate('blur');
+      userEvent.hover(await screen.findByTestId('arithmeticErrorWarning'));
+      const tooltipEl = await screen.findByText('Division by 0 is not allowed');
+      expect(tooltipEl).toBeInTheDocument();
 
-      expect(newWrapper.find('QueryField ArithmeticError').prop('title')).toBe(
-        'Maximum operators exceeded'
-      );
+      const input = screen.getAllByRole('textbox')[0];
+      expect(input).toHaveValue('1 / 0');
+
+      userEvent.type(input, '{selectall}1+1+1+1+1+1+1+1+1+1+1+1');
+
+      userEvent.hover(await screen.findByTestId('arithmeticErrorWarning'));
+      waitFor(() => {
+        expect(tooltipEl).toHaveTextContent('Maximum operators exceeded');
+      });
```

Comment:
This part is giving me some `act()` troubles, will have to look into it tomorrow.